File: bee1117.py
quant = 0
total = 0
while True:
    nota = float(input())
    if nota < 0.0 or nota > 10.0:
        print("nota invalida")
    else:
        total += nota
        quant += 1
    if quant == 2:
        media = total / quant
        break
print("media =",media)

File: bee1117.c
#include <stdio.h>
 
int main() {
 
    float nota,total = 0, media = 0;
    int quant = 0;
    while(1==1){
        scanf("%f",&nota);
        if(nota < 0.0 || nota > 10.0){
            printf("nota invalida\n");
        }
        else{
           total = total + nota;
           quant ++;
        }
        if(quant == 2){
            media = total / quant;
            break;
        }
        
    }
    printf("media = %.2f",media);
 
    return 0;
}